src/base/rss/rss_article.h

```diff
@@ -46,9 +46,6 @@ namespace RSS
 
         friend class Feed;
 
-        Article(Feed *feed, const QVariantHash &varHash);
```

Comment:
I would still prefer to have constructors at top of class declaration (even private ones).

Reply:
I have other opinions but I don't insist, so it is reverted.